python/mxnet/ndarray/numpy/_op.py

```diff
@@ -5308,3 +5308,67 @@ def nan_to_num(x, copy=True, nan=0.0, posinf=None, neginf=None, **kwargs):
         return _npi.nan_to_num(x, copy=copy, nan=nan, posinf=posinf, neginf=neginf, out=None)
     else:
         raise TypeError('type {} not supported'.format(str(type(x))))
+
+
+@set_module('mxnet.ndarray.numpy')
+def where(condition, x, y):
```

Comment:
We need to support the case where there is only one input. You can do this for the imperative mode only by falling back to `condition.nonzero()`.